src/components/FilterPane/FilterPane.jsx

```diff
@@ -197,12 +212,24 @@ const FilterPane = ({ mermaidUserData }) => {
             </Box>
           )}
         >
-          {countries.map((country) => (
+          {remainingDisplayedCountries.length > 0 ? (
```

Comment:
Is `> 0` necessary here?

Reply:
It's not necessary, I just wanted to be explicit. I can remove it
It can be but ternary is better because if remainingDisplayedCountries ever has a length of 0, it will render '0' instead of nothing https://www.crocoder.dev/blog/react-conditional-rendering/#:~:text=However%2C%20there%20is%20a%20catch,render%20exactly%20what%20you%20expect.

Reply:
Makes sense.
I'm more used to the `!!` alternative but whatever works for you.